netbox/extras/conditions.py

```diff
@@ -44,7 +44,8 @@ class Condition:
         bool: (EQ, CONTAINS),
         int: (EQ, GT, GTE, LT, LTE, CONTAINS),
         float: (EQ, GT, GTE, LT, LTE, CONTAINS),
-        list: (EQ, IN, CONTAINS)
+        list: (EQ, IN, CONTAINS),
+        type(None): (EQ)
```

Comment:
I think this needs the comma to be evaluated as a tuple. It's not causing a problem right now because `'eq' in 'eq'` evaluates to True.
```suggestion
        type(None): (EQ,)
```